README.md

```diff
@@ -0,0 +1,20 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# Helm charts for Apache Ozone
+
+This repository will provide Helm charts for installing Apache Ozone on Kubernetes.
```

Comment:
```suggestion
# Running Ozone in Kubernetes

This repository will provide Helm charts for installing Apache Ozone on Kubernetes. This repo can be expanded to include other ways to run Ozone in Kubernetes including a native operator.
```

Reply:
Let's keep this repo dedicated for the Helm chart.